cmd/bindings/bindings.go

```diff
@@ -76,7 +76,9 @@ func cmdBindingsList(b *bindings, printf shared.FormatFn) *cobra.Command {
 		Args:  cobra.NoArgs,
 
 		Run: func(cmd *cobra.Command, _ []string) {
-			b.cmdList(printf)
+			if err := b.cmdList(printf); err != nil {
+				return
```

Comment:
For the functions like these, we should probably just return the error. Just change the `Run:` to `RunE:` and the internal code to `return b.cmdList(printf)` instead of doing the if check.

Reply:
Good to know! Will correct it shortly.